tests/pos/Annotations.scala

```diff
@@ -0,0 +1,8 @@
+package foo.bar
+
+import jdk.jfr.Enabled
+
+@Enabled
+@Deprecated
```

Comment:
Shouldn't the test involve using the same annotation multiple times to trigger use of RepeatableAnnotations, like in https://github.com/scala/scala/commit/954c5d32d71a43b141be546877b01183a994a1b2#diff-8bd66281732306fbd83125f4c1bba7a81797b9ed31a0857afb0b805dd6e1fe6cR258-R262 ?

Reply:
I guess that would *also* be useful to test, though it wasn't the scenario I was aiming to fix: because the 'annotations.groupBy(_.symbol)` doesn't have a predictable order of keys, even regular 'unrepeated' annotations would not necessarily end up in the bytecode in the same order.

Reply:
Ah, I didn't realize that this code wasn't simply bypassed when no repeated annotations exist. But yeah, if you could copy the tests that are in the scala 2 PRs that would be nice: to compile multiple files at once (the .java annotation definition and the scala usage) you can create a subdirectory in tests/pos.

Reply:
Gotcha, added!